packages/synapse-react-client/src/components/SynapseHomepageV2/SynapseHotDropsBackground.tsx

```diff
@@ -0,0 +1,157 @@
+import { Box } from '@mui/material'
+
+const SynapseHotdropsBackground = () => {
+  return (
+    <Box
+      sx={{
+        position: 'absolute',
+        top: 0,
+        left: 0,
+        zIndex: -1,
+        width: '100%',
+        height: '100%',
+        overflow: 'hidden',
+        backgroundColor: '#121212',
+        '& svg': {
+          width: '100%',
+          height: '100%',
+        },
+        '@keyframes float1': {
+          '0%, 100%': { transform: 'translateY(0px) scale(1)' },
+          '50%': { transform: 'translateY(11px) scale(1.02)' },
+        },
+        '@keyframes float2': {
+          '0%, 100%': { transform: 'translateY(0px) scale(1)' },
+          '50%': { transform: 'translateY(17px) scale(1.03)' },
+        },
+        '@keyframes float3': {
+          '0%, 100%': { transform: 'translateY(0px) scale(1)' },
+          '50%': { transform: 'translateY(23px) scale(1.15)' },
+        },
+        '@keyframes float4': {
+          '0%, 100%': { transform: 'translateY(0px) scale(1)' },
+          '50%': { transform: 'translateY(13px) scale(1.05)' },
+        },
+        '@keyframes float5': {
+          '0%, 100%': { transform: 'translateY(0px) scale(1)' },
+          '50%': { transform: 'translateY(15px) scale(1.06)' },
+        },
+      }}
+    >
+      <svg
```

Comment:
if this SVG isn't using any JavaScript, I'd prefer it is put in a different file that can be previewed by IDEs (and might handled differently by a bundler)